.github/workflows/wheels-dependencies.sh

```diff
@@ -122,31 +122,37 @@ function build_libavif {
         build_simple nasm 2.16.03 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03
     fi
 
-    # For rav1e
-    curl https://sh.rustup.rs -sSf | sh -s -- -y
-    . "$HOME/.cargo/env"
     if [ -z "$IS_ALPINE" ] && [ -z "$SANITIZER" ] && [ -z "$IS_MACOS" ]; then
         yum install -y perl
         if [[ "$MB_ML_VER" == 2014 ]]; then
             yum install -y perl-IPC-Cmd
         fi
     fi
 
+    local build_type=MinSizeRel
+
+    if [[ -z "$IS_ALPINE" ]] && [[ "$MB_ML_VER" == 2014 ]]; then
+        build_type=Release
+    fi
+
     local out_dir=$(fetch_unpack https://github.com/AOMediaCodec/libavif/archive/refs/tags/v$LIBAVIF_VERSION.tar.gz libavif-$LIBAVIF_VERSION.tar.gz)
+    # CONFIG_AV1_DECODER=0 is a flag for libaom (included as a subproject of
+    # libavif) to disable the compilation and inclusion of aom's AV1 decoder.
     (cd $out_dir \
         && CMAKE_POLICY_VERSION_MINIMUM=3.5 cmake \
             -DCMAKE_INSTALL_PREFIX=$BUILD_PREFIX \
             -DCMAKE_INSTALL_LIBDIR=$BUILD_PREFIX/lib \
-            -DCMAKE_BUILD_TYPE=Release \
-            -DBUILD_SHARED_LIBS=OFF \
+            -DCMAKE_INSTALL_NAME_DIR=$BUILD_PREFIX/lib \
+            -DCMAKE_MACOSX_RPATH=OFF \
```

Comment:
What was the thinking behind `CMAKE_MACOSX_RPATH=OFF`?

Reply:
Without it, delocate fails on libavif.dylib

Reply:
It looks as though my delocate `@rpath` error was fixed by the addition of `-DCMAKE_INSTALL_LIB_DIR`, making this change unnecessary